### PR TITLE
artist adapter, model, serializer, dummy

### PR DESCRIPTION
**Pull request by djvoa12**
Initial Artist API Methods:
- biographies (`biography` payload)
- blogs (`blog` payload)
- familiarity (`artist` payload)

Updated `README.md`

Unit Tests:
- Adapter

### CODE REVIEW

**Comment by elwayman02** [comment]
bio.trucated?shouldn't that be truncated?

**Comment by elwayman02** [comment]
Do you think we should be explicit or implicit with type casting on attr()? I am not sold on either path... It's nice to have the conciseness but I'm not sure if there is enough advantage over the explicit declaration, which may have less room for error. Thoughts?

**Comment by elwayman02** [comment]
We could probably write a common method on a base serializer that handles the camelization more dynamically so we don't have to be explicitly converting every instance that comes up

**Comment by elwayman02** [comment]
Though that could get complicated with embedded objects...

**Comment by djvoa12** [comment]
1) Yes, I missed that (bio.truncated). Will fix.
2) I tend to explicitly declare data-type on primitive values. If it is an array or POJO, I would leave it empty. One potential value would be that new developers would be able to immediately tell what they are suppose to be.
3) Nested objects could be an issue, but maybe we can limit it to one nested level. I would have to look at what the payloads are, but none of them should be complicated.

File: tests/dummy/app/templates/artists/familiarity.hbs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<table>
+  <tr>
+    <th>Artist</th>
+    <th>Familiarity Rating</th>
+  </tr>
+  {{#each model as |artist|}}
+    {{log artist}}
```
Comment (elwayman02) [inline tests/dummy/app/templates/artists/familiarity.hbs]: remove this?

File: addon/serializers/echonest.js [hunk 2]
```diff
@@ -1,26 +1,71 @@
...
+                    }
+                    if (isPresent(removalList)) {
+                      item = this.removeProperty(item);
+                    }
+
+                    return item;
+                });
+                return this._super(store, primaryModelClass, { [`echonest-${key}`]: items }, id, requestType);
+            } else if (isPresent(payload.response[key])) {
+                return this._super(store, primaryModelClass, { [`echonest-${key}`]: [ payload.response[key] ] }, id, requestType);
```
Comment (djvoa12) [inline addon/serializers/echonest.js]: I copied your latest parent serializer. The payload needed to be in an array so I added that part on this line.

File: addon/adapters/echonest-blog.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import ArtistAdapter from './echonest-artist';
+
+export default ArtistAdapter.extend();
```
Comment (elwayman02) [inline addon/adapters/echonest-blog.js]: nice, smart!

**Pull request merged**